server/views/pages/vary-approve/view.njk

```diff
@@ -2,10 +2,6 @@
 
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
-
-{% from "partials/additionalLicenceConditionsSummary.njk" import additionalLicenceConditions %}
-{% from "partials/additionalPssConditionsSummary.njk" import additionalPssConditions %}
-{% from "partials/bespokeConditionsSummary.njk" import bespokeConditions %}
 {% from "partials/variedConditions.njk" import variedConditions %}
```

Comment:
Should bespokeConditions not still be imported?

Reply:
These imports are not used in the file.